pkg/backupapi/file.go

```diff
@@ -24,3 +35,39 @@ type Part struct {
 	Size       int    `json:"size"`
 	Etag       string `json:"etag"`
 }
+
+// UploadFile uploads given file to server.
+func (c *Client) UploadFile(fn string, r io.Reader) error {
```

Comment:
Can you introduce a function upload file using multipart endpoint API?

Reply:
Should be added later

Reply:
ok